topics/proteomics/tutorials/peptide-library-data-analysis/tutorial.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: tutorial_hands_on
+
+title: Peptide Library Data Analysis
+level: Intermediate
+questions:
+  - How to utilize quantitative properties of amino acids and peptide sequence to analyse peptide data?
+objectives:
+  - Calculate descriptors
+  - Quantitative analysis of peptide sequence properties
+time_estimation: '20m'
+contributors:
+   - jaidevjoshi83
+   - blankenberg
+---
+
+# Introduction
+{:.no_toc}
+
+
+Several computational methods have been proven very useful in the initial screening and prediction of peptides for various biological properties. These methods have emerged as effective alternatives to the lengthy and expensive traditional experimental approaches. In this tutorial, we will be discussing how to analyze peptide libraries on the basis of quantitative properties. In this tutorial, we will learn how to use different utilities of PDAUG to calculate various peptide-based features and utilize these features for various informative plots.
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+## Peptide Data
+
+In this step, we will retrieve the inbuild dataset, which contains anti-microbial (AMPs) and transmembrane peptides (TMPs).
+
+
+> ### {% icon hands_on %} Hands-on: Fetching inbuild data
+> 
+> 1. {% tool [PDAUG Peptide Data Access](toolshed.g2.bx.psu.edu/repos/jay/pdaug_peptide_data_access/pdaug_peptide_data_access/0.1.0) %} with the following parameters:
+>    - *"Datasets"*: `AMPvsTM` 
+>
+{: .hands_on}
+
+
```

Comment:
Why does the output of PDAUG peptide data access have a target column with 0 and 1; why not AMP and TM, why is it called target, to my understanding the column indicates the peptide type (AMP or TMP). Later it is called class label 0 and 1, which makes it even more confusing.

Reply:
Class label and target variable both represent the dependent variable that we predict during ML model building.  Several algorithms and libraries that implement ML algorithms do not support string or text as a target variable, so the default class labels (target variable) are provided as 0 and 1 for two groups of peptides. However, in this workflow, we can use AMPs and TMPs as class labels (target variable) without any problem. Class labels have been changed as AMPs and TMPs instead of 0 and 1.